src/GDGUkraine/rest_controller.py

```diff
@@ -511,6 +511,23 @@ def generate_invites(self, id):
             raise HTTPError(500, "Cannot save generated invites") from e
         return {"ok": True}
 
+    @cherrypy.tools.json_out()
+    @cherrypy.tools.authorize()
+    def record_visit(self, id):
+        '''POST /api/events/:id/check-in'''
+        req = cherrypy.request
+        orm_session = req.orm_session
+        reg_id = int(id)
+        reg_data = api.get_event_registration_by_id(orm_session, reg_id)
+        if not reg_data:
+            raise HTTPError(400,
```

Comment:
Actually, I don't really know if situation with 'Not Found' registration could possibly exist. If I am not mistaken, reg_id should be valid and inherited from layer above. Thus the ways reg_data is not valid are:
1) someone changed/deleted DB record for this participant between time when 'register' button appeared and pressed (which, as I understand, will be the source of id in request)
2) code exception
3) [non-realistic] someone put his own self-created id into request
Please, correct me if I'm wrong.
P.S. @anxolerd me too;)